pages.sv/common/yarn-why.md

```diff
@@ -0,0 +1,8 @@
+# yarn-why
```

Comment:
This page is outdated in the primary (English) pages because `yarn` (v1 or v2) has a built-in subcommand `yarn why` to know this information. https://classic.yarnpkg.com/en/docs/cli/why

The translation is OK, but feel free to send an update to the English page about this.